file-formats/clas/type/zif_aff_clas_v1.intf.abap

```diff
@@ -21,6 +21,9 @@ INTERFACE zif_aff_clas_v1 PUBLIC.
       "! <p class="shorttext">Behavior class (behavior implementations)</p>
       "! Behavior class (behavior implementations)
       behavior_class                 TYPE ty_category VALUE '06',
+      "! <p class="shorttext">Event handler class</p>
+      "! Event handler class
+      event_handler                  TYPE ty_category VALUE '07',
```

Comment:
"Event handler class" seems to be too generic. I double check it with some colleagues

Reply:
I have talked to some colleagues. They suggested to call this category `entity_event_handler` to indicate it is an "entity event handler class (pool)".